ratpack-bearer-auth/src/main/java/st/ratpack/auth/PredicateValidatorTuple.java

```diff
@@ -0,0 +1,28 @@
+package st.ratpack.auth;
+
+import ratpack.exec.Promise;
+
+import java.util.Optional;
+import java.util.function.Predicate;
+
+public class PredicateValidatorTuple {
+    private Predicate<String> predicate;
```

Comment:
temporarily copied @beckje01 's PredicateValidatorTuple for testing purpose, until https://github.com/SmartThingsOSS/ratpack-bearer-auth/pull/37 is merged